contracts/ConvergentCurvePool.sol

```diff
@@ -538,16 +529,14 @@ contract ConvergentCurvePool is IMinimalSwapInfoPool, BalancerPoolToken {
         }
     }
 
-    /// @dev Burns at least enough LP tokens from the sender to produce
-    ///      as set of minium outputs.
-    /// @param minOutputUnderlying The minimum output in underlying
-    /// @param minOutputBond The minimum output in the bond
+    /// @dev Burns a number of LP tokens and returns the percent of the pool
+    ///      they represent
```

Comment:
Maybe it is just me, but this comment confuses me.  Specifically `returns the percent of the pool they represent`.